emspy/query/fltquery.py

```diff
@@ -493,47 +533,59 @@ def _boolean_filter(op, d):
 def _discrete_filter(op, d, flt):
 	fld_info = d[0]
 
-	if op in list(basic_ops.keys()):
-		# Single input basic coniditonal operation
-		t_op = basic_ops[op]
-		val_info = d[1]
-		vid  = flt.get_value_id(val_info['value'], field_id = fld_info['value'])
-		val_info['value'] = vid
-		fltr = _filter_fmt1(t_op, fld_info, val_info)
-
-	elif op in [" in ", " not in "]:
-		t_op = sp_ops[op]
-		val_list = [{'type':x['type'], 'value':flt.get_value_id(x['value'], field_id=fld_info['value'])}\
-					 for x in d[1:]]
-		inp = [fld_info] + val_list
-		fltr = _filter_fmt1(t_op, *inp)
+	if not isinstance(op, list):
```

Comment:
What's the reason for checking if op is a list?  Was that happening before?  Or is that new with between filtering? 

Do you think this should be split out to reduce indentation?  In other words, check the type first, raise a ValueError if it's wrong, and then continue? 

```python
if not isinstance(op, list):
    raise ValueError("")

if op in list(basic_op.keys()):
   ...
```